cmd/erasure-object.go

```diff
@@ -159,6 +160,15 @@ func (er erasureObjects) GetObjectNInfo(ctx context.Context, bucket, object stri
 
 	fi, metaArr, onlineDisks, err := er.getObjectFileInfo(ctx, bucket, object, opts)
 	if err != nil {
+		if errors.Is(err, errFileNotFound) ||
+			errors.Is(err, errFileVersionNotFound) {
+			// proxy to replication target if active-active replication is in place.
+			reader, proxy := proxyGetToReplicationTarget(ctx, bucket, object, rs, h, opts)
```

Comment:
It doesn't make sense to do this at the handler level because GetObjectNInfo is also used by select api - we would need to proxy wherever GetObjectNInfo is called. Replication is only relevant to erasure layer, hence this approach